configuration/jsr223.md

```diff
@@ -14,7 +14,13 @@ and are comfortable working with the command line prompt of the operating system
 
 [JSR223](https://docs.oracle.com/javase/8/docs/technotes/guides/scripting/) ([spec](https://jcp.org/aboutJava/communityprocess/pr/jsr223/index.html)) is a standard scripting API for Java Virtual Machine (JVM) [languages](https://en.wikipedia.org/wiki/List_of_JVM_languages).
 The JVM languages provide varying levels of support for the JSR223 API and interoperability with the Java runtime.
-Currently the following languages are known to work well for openHAB scripting: [**Apache Groovy**](/addons/automation/groovyscripting/) (JVM scripting language), [**JavaScript**](/addons/automation/jsscripting/), [**Ruby**](/addons/automation/jrubyscripting/), and [**Jython**](https://www.openhab.org/addons/automation/jythonscripting/) (Python on the JVM).
+Currently the following languages are known to work well for openHAB scripting: [**Apache Groovy**](/addons/automation/groovyscripting/) (JVM scripting language), **JavaScript/ECMA - 262 Edition 5.1**, [**JavaScript/ECMAScript 262 Edition 11**](/addons/automation/jsscripting/), [**Ruby**](/addons/automation/jrubyscripting/), and [**Jython**](https://www.openhab.org/addons/automation/jythonscripting/) (Python on the JVM).
+
+Please note that **JavaScript/ECMA - 262 Edition 5.1** (whose setup is described here) is using an older engine called NashornJS, which has been removed from JDK 15 and will be unavailable one day. 
+Moreover, you need to work with the raw openHAB Java APIs in this JavaScript environment which is not easy.
```

Comment:
This too is not entirely correct. There is a Helper Library for JS. If the problem is that you have to install that library via means other than JRuby, Jython, and Groovy (which does lack a helper library) also should be listed. 

Over all I think it's premature to take a stand on which language to use over all (if we ever get to that point) but I also think it's important to stere those who want to use JS to use JS Scripting instead of Nashorn.

So maybe rework this section a little bit along the lines of:

> Please note that **JavaScript/ECMA - 262 Edition 5.1** which is referenced here is using an older engine called NashornJS.
> This version of ECMAScript is end of life and lacks many important features that have been added to the language since 2009.
> If you plan to use ECMAScript as your rules language, please use [**JavaScript/ECMAScript 262 Edition 11**](/addons/automation/jsscripting/), also known as [JS Scripting](/addons/automation/jsscripting/) which includes the latest **ES6** language features, supports third party node libraries, and comes with a well documented helper library that makes interacting with all the APIs documented on this page much simpler and straight forward.

I think that makes it a bit more clear that we are not pushing someone who loves Ruby, for example, towards using JS Scripting but instead limiting the advice to those who are drawn to JS in the first place.

Reply:
> This too is not entirely correct. There is a Helper Library for JS. If the problem is that you have to install that library via means other than JRuby, Jython, and Groovy (which does lack a helper library) also should be listed.

Out of interest: Where can I find those helper libraries? Do you refer to https://openhab-scripters.github.io/openhab-helper-libraries/JavaScript/Reference.html?

Reply:
Okay thanks, I remember having a look at this helper library some time ago. 
One thing that I missed back then was the documentation for that one.